sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -25,6 +25,12 @@
 - Removed `RequestId` from the return types.
 - Changed `BodyStream` parameter of `UploadRange` function from pointer to reference.
 - Removed `PreviousShareSnapshot` from `GetShareFileRangeListOptions`, use `ShareFileClient::GetRangeListDiff` instead.
+- Removed `PathRenameMode`.
+- Removed `PathGetPropertiesAction`.
+- Removed `PathSetAccessControlRecursiveMode`.
+- Removed `FileSystemResourceType`.
+- Removed `FileSystemResourceType`.
+- Removed `IsAccessTierInferred` and `AccessTierChangedOn` from `PathProperties`.
```

Comment:
I am curious, how come we can remove these from `PathProperties` (it's fine that we are, just wondering)? Why did we have them to begin with?

Reply:
Access Tier related things are not available for datalake service. Jinming and I had a discussion and decide that we should remove them. However this API calls blob API, which will return them sometimes. That's why we originally have those fields.